client.js

```diff
@@ -570,7 +570,10 @@ async function instanceManagement(instanceconfig) {
     console.log("Started instanceManagement();");
 
     /* Open websocket connection to master */
-	var socket = ioClient("http://"+config.masterIP+":"+config.masterPort);
+	var socket = ioClient("http://"+config.masterIP+":"+config.masterPort+"?token="+config.masterAuthToken);
```

Comment:
Shouldn't this use the updated masterful already? We merged that config change already

Reply:
This PR is against the 1.2.x branch if you haven't noticed.

Reply:
Oh, sorry. But then this is a major break, we need a 1.3 release then. This type of fix should not be done in patch versions.